src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,39 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int FIELDS_MIN_LENGTH = 6;
+    private static final int USER_MIN_AGE = 18;
+
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
```

Comment:
Please introoduce some methods to make code more readable

Reply:
What do you mean by " introoduce some methods "?
 
 I am planning to divide into checking for nulls, checking fields length and other part